SoftLayer/managers/account.py

```diff
@@ -70,21 +71,32 @@ def get_upcoming_events(self, event_type):
             }
         }
 
-        self.add_event_filter(_filter, event_type)
+        self.add_event_filter(_filter, event_type, date_min)
 
         return self.client.call('Notification_Occurrence_Event', 'getAllObjects', filter=_filter, mask=mask, iter=True)
 
     @staticmethod
-    def add_event_filter(_filter, event_type):
+    def add_event_filter(_filter, event_type, date_min):
```

Comment:
Set a default here so you don't have do update ever function that calls this with the `date_min=""` 
```suggestion
    def add_event_filter(_filter, event_type, date_min=None):
```

Reply:
Done